basics/json-validation/README.md

```diff
@@ -1,8 +1,8 @@
 This is a contrived example intended to illustrate a few important actix-web features.
 
 *Imagine* that you have a process that involves 3 steps.  The steps here
-are dumb in that they do nothing other than call an
-httpbin endpoint that returns the json that was posted to it.  The intent here
+are dumb in that they do nothing other than call a http bin endpoint that
```

Comment:
I'm not an expert in English, but since 'h' in "http" is pronounced as /ˈeɪtʃ/, I guess it should be "an".